### ISSUE TITLE
1.25 zip structure has changed

### ISSUE THREAD

**Issue by CelesteGH** [Genymobile/scrcpy#3634]
Hi

1.25 zip assets now have a sub folder with the same name as the zip.
Is this a mistake or how all zipped assets after 1.25 will be ?

This is for auto updating purposes.

Thanks in advance.

**Comment by rom1v** [comment]
Yes, sorry I changed that without notice: https://github.com/Genymobile/scrcpy/commit/8b38b11875eee81186adebe709ef0fcfd68d88ab

I think it is a good practice to have a parent directory in a zip file.
